tests/unittests/tests-gcoap/tests-gcoap.c

```diff
@@ -175,7 +175,7 @@ static void test_gcoap__client_put_req_overfill(void)
 }
 
 /*
- * Builds on get_req test, to test gcoap_add_qstring() to add Uri-Query
+ * Builds on get_req test, to test coap_opt_add_uqery() to add Uri-Query
```

Comment:
```suggestion
 * Builds on get_req test, to test coap_opt_add_uquery() to add Uri-Query
```

Reply:
Thanks for the fix/suggestion. Let's see if we retain this based on my comment below.